src/Microsoft.ML.Data/Transforms/ColumnCopying.cs

```diff
@@ -31,8 +31,22 @@
 namespace Microsoft.ML.Transforms
 {
     /// <summary>
-    /// <see cref="ColumnCopyingEstimator"/> copies the input column to another column named as specified in the parameters of the transformation.
+    /// Estimator creating the <see cref="ColumnCopyingTransformer"/>.
```

Comment:
>Estimator [](start = 8, length = 9)

Are we doing cref=IEstimator like in trainers, or is that too much?

Reply:
I prefer the wording in the trainers i.e. "The <see cref="IEstimator{TTransformer}"/> for the XXX transformer." (Not fussed about whether we cref the Estimator or not).

I think using *creating* here is confusing, as we create the estimator object using the extension method.